src/rpc/blockchain.cpp

```diff
@@ -2293,15 +2297,19 @@ UniValue scantxoutset(const JSONRPCRequest& request)
         g_scan_progress = 0;
         int64_t count = 0;
         std::unique_ptr<CCoinsViewCursor> pcursor;
+        CBlockIndex* tip;
         {
             LOCK(cs_main);
             ::ChainstateActive().ForceFlushStateToDisk();
             pcursor = std::unique_ptr<CCoinsViewCursor>(pcoinsdbview->Cursor());
-            assert(pcursor);
+            tip = ::ChainActive().Tip();
+            assert(pcursor && tip);
```

Comment:
Could leave them in separate lines, so that an assert failure is more verbose and helpful?

Reply:
Done.